client/src/app/components/tool-container/tool-container.component.scss

```diff
@@ -0,0 +1,21 @@
+@import "~bootstrap/scss/mixins";
+@import "~bootstrap/scss/_variables";
+@import "~bootstrap/scss/_forms";
+@import "~bootstrap/scss/_input-group";
+@import "~bootstrap/scss/_tooltip";
+@import "~bootstrap/scss/_buttons";
+@import "~bootstrap/scss/_button-group";
+@import "~bootstrap/scss/_close";
```

Comment:
For reference, you can (should) just uncomment these imports in [_bootstrap-pack.scss ](https://github.com/rabix/cottontail/blob/next/client/src/assets/sass/_bootstrap-pack.scss)